src/main/java/org/onosproject/monitoring/server/IntPushGateway.java

```diff
@@ -0,0 +1,240 @@
+package org.onosproject.monitoring.server;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.channel.socket.DatagramPacket;
+import io.prometheus.client.CollectorRegistry;
+import io.prometheus.client.Gauge;
+import io.prometheus.client.exporter.PushGateway;
+import org.onosproject.monitoring.packet.*;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.HashMap;
+
+
+//import java.net.DatagramPacket;
+
+public class IntPushGateway {
```

Comment:
We have double work on that. Please see https://github.com/serkantul/prometheus_int_exporter/blob/serkantul-prometheus-integration/src/main/java/io/prometheus/client/exporter/INTExporter.java